stories/va-select.stories.jsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import { getWebComponentDocs, StoryDocs, propStructure } from './wc-helpers';
+
+const selectDocs = getWebComponentDocs('va-select');
+
+export default {
+  title: 'Components/va-select',
+  parameters: {
+    docs: {
+      /* eslint-disable-next-line react/display-name */
+      page: () => <StoryDocs docs={selectDocs.docs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  label: 'Branch of Service',
+  name: 'branch',
+  value: 'army',
+  required: false,
+  error: null,
+  ariaLiveRegionText: 'You selected',
+  options: [
+    <option key="1" value="navy">
+      Navy
+    </option>,
+    <option key="2" value="army">
+      Army
+    </option>,
+    <option key="3" value="marines">
+      Marines
+    </option>,
+    <option key="4" value="air-force">
+      Air Force
+    </option>,
+    <option key="5" value="coastguard">
+      Coastguard
+    </option>,
+  ],
+};
+
+const Template = ({
+  label,
+  name,
+  value,
+  required,
+  error,
+  ariaLiveRegionText,
+  options,
+}) => {
+  const [modifiedOptions, setModifiedOptions] = useState(options);
+
+  return (
+    <>
+      <button
+        onClick={() => {
+          console.log(modifiedOptions);
+          setModifiedOptions([
+            ...modifiedOptions,
+            <option key="6" value="new">
+              Something new
+            </option>,
+          ]);
+        }}
+      >
+        Change the options
+      </button>
```

Comment:
Let's move this to a story rather than the template. We want to demonstrate the behavior of changing the options dynamically, but we don't need to do that for every story. It kind of detracts from showcasing the component itself.

Reply:
Due to the state variables, I left this in the template but it's conditionally rendered based on the arguments. What do you think about that?